Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
In an ideal case, start it, test it, stop it. If not, checking any message from e.g. `orientdb-console` would be good (redirecting stderr with `2>&1 if necessary)